### ISSUE TITLE
Message checkboxes can only be enabled holding down the control key

### ISSUE THREAD

**Issue by sukiletxe** [UnigramDev/Unigram#2881]
**Describe the bug**

When one hears that there is a checkbox,he / she expects that it is toggled with space. This doesn't happen with the message selection ones.

**To Reproduce**

1. With Narrator enabled, open a chat.
2. Right click on a message (or press Shift + F10 or the applications key, the result is the same). The context menu opens.
3. Enable the "select" option.
4. Move to another message with the arrow keys and try toggling it with the space bar.
5. Nothing happens.

**Expected behavior**

The checkbox should be toggled, as no other focusable item (link, mention, message author, original message) was focussed other than the message.

**Version Info**
 - Unigram: Unigram 9.0 (7840) X64 Direct
 - Windows: Windows 10 Versión 21H2 (compilación de SO 19044.2130)

**Comment by AbyssallArmy884** [comment]
Sambungkan telegram bot

**Comment by amirmahdifard** [comment]
this is already been resolved: selecting the first message by the select option and checking the others just with space works perfectly and as expected: should be closed as completed.

**Comment by sukiletxe** [comment]
Strange, not for me.

**Comment by amirmahdifard** [comment]
@sukiletxe yeah for me it perfectly works, if you don't beleave me i can even record my screenreader and send you.

**Comment by sukiletxe** [comment]
I am using version 10.2.1 (9167), in:

Edición	Windows 10 Home

Versión	22H2

Compilación del sistema operativo	19045.3570

Experiencia	Windows Feature Experience Pack 1000.19052.1000.0

I have tried in user chats, groups and channels, and I still have to press control to select messages. What are you using? In which chats does it work?
